File: app/src/main/java/com/wealoha/social/api/user/Profile2Data.java
package com.wealoha.social.api.user;

import java.util.Map;

import com.wealoha.social.api.common.dto.ImageDTO;
import com.wealoha.social.api.user.dto.UserDTO;
import com.wealoha.social.beans.ResultData;

public class Profile2Data extends ResultData {

	public UserDTO user;
	public Map<String, ImageDTO> imageMap;
	public boolean friend;
	public boolean liked;
}

File: app/src/main/java/com/wealoha/social/api/user/Profile2Data.kt
package com.wealoha.social.api.user

import com.wealoha.social.api.common.dto.ImageDTO
import com.wealoha.social.api.user.dto.UserDTO
import com.wealoha.social.beans.ResultData

class Profile2Data : ResultData() {
    var user: UserDTO? = null
    var imageMap: Map<String, ImageDTO>? = null
    var friend = false
    var liked = false
}